api/db-cdr/generate-cdr/build-review-all-events.sh

```diff
@@ -276,4 +293,32 @@ LEFT JOIN \`$BQ_PROJECT.$BQ_DATASET.concept\` c1 on a.PROCEDURE_CONCEPT_ID = c1.
 LEFT JOIN \`$BQ_PROJECT.$BQ_DATASET.concept\` c2 on a.PROCEDURE_SOURCE_CONCEPT_ID = c2.CONCEPT_ID
 left join \`$BQ_PROJECT.$BQ_DATASET.visit_occurrence\` v on a.VISIT_OCCURRENCE_ID = v.VISIT_OCCURRENCE_ID
 left join \`$BQ_PROJECT.$BQ_DATASET.concept\` c3 on v.visit_concept_id = c3.concept_id
-JOIN \`$BQ_PROJECT.$BQ_DATASET.person\` p on a.PERSON_ID = p.PERSON_ID"
\ No newline at end of file
+JOIN \`$BQ_PROJECT.$BQ_DATASET.person\` p on a.PERSON_ID = p.PERSON_ID"
+}
+
+if [[ "$DATA_BROWSER" == true ]]
+then
+  echo "Building index for data browser. Skipping creation of the cb_review_all_events table."
+  exit 0
+fi
```

Comment:
This is no longer needed cause of the conditional workflows.

Reply:
Done.